packages/runtime/src/prop-controllers/copy.ts

```diff
@@ -0,0 +1,15 @@
+import { Descriptor } from './descriptors'
+import { copy as BackgroundsCopy } from './copy/Backgrounds'
+
+function defaultCopy(props: any) {
+  return props
+}
+
+export function getCopyFunction(descriptor: Descriptor) {
+  switch (descriptor.type) {
+    case 'Backgrounds':
+      return BackgroundsCopy
+    default:
+      return defaultCopy
+  }
+}
```

Comment:
Don't have a strong opinion as to the folder structure of how we manage these copy functions.

Although, definitely 1 copy function per file. They're a bit big. You might know if there's a more natural place for them to reside though